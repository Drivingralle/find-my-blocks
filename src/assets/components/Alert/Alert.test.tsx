import React from "react";
import renderer from "react-test-renderer";
import { Alert } from ".";

it("renders the Alert", () => {
  const tree = renderer.create(
    <Alert>Hello World</Alert>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders the right classes for success/warning/error", () => {
  const success = renderer.create(
    <Alert type="success">Hello World</Alert>
  ).toJSON();

  const warning = renderer.create(
    <Alert type="warning">Hello World</Alert>
  ).toJSON();

  const error = renderer.create(
    <Alert type="error">Hello World</Alert>
  ).toJSON();

  expect(success).toMatchSnapshot();
  expect(warning).toMatchSnapshot();
  expect(error).toMatchSnapshot();
});
