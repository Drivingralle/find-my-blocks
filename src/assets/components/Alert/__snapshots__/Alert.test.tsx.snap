// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Alert 1`] = `
<div
  className="alert"
>
  Hello World
</div>
`;

exports[`renders the right classes for success/warning/error 1`] = `
<div
  className="alert success"
>
  Hello World
</div>
`;

exports[`renders the right classes for success/warning/error 2`] = `
<div
  className="alert warning"
>
  Hello World
</div>
`;

exports[`renders the right classes for success/warning/error 3`] = `
<div
  className="alert error"
>
  Hello World
</div>
`;
